docs/apache-airflow-providers/index.rst

```diff
@@ -119,6 +132,43 @@ When you write your own provider, consider following the
 `Naming conventions for provider packages <https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#naming-conventions-for-provider-packages>`_
 
 
+Community-managed vs. 3rd party providers
+'''''''''''''''''''''''''''''''''''''''''
+
+While you can develop your own providers, Apache Airflow has 60+ providers that are managed by the community.
+They are part of the same repository as Apache Airflow (we use ``monorepo`` approach where different
+parts of the system are developed in the same repository but then they are packaged and released separately).
+All the community-managed providers are in 'airflow/providers' folder and they are all sub-packages of
+'airflow.providers' package. All the providers are available as ``apache-airflow-providers-<PROVIDER_ID>``
+packages.
+
+The capabilities of the community-managed providers are the same as the third-party ones are the same. When
+the providers are installed from PyPI, they provide the entry-point containing the metadata as described
+in the previous chapter. However when they are locally developed, together with Airflow, the mechanism
```

Comment:
This document is user-facing, so talking about installing providers from sources feels out of place here.

Reply:
Good point. I will make a separate CUSTOM_PROVIDERS.rst document.

Reply:
Actuallly moved this content to CONTRIBUTING.rst.